PORTAL/jobs/_pyperformance.py

```diff
@@ -483,15 +566,16 @@ def _match(self, specifier, checksuite):
         #    return True
         return False
 
-    def _match_version(self, version):
+    def _match_version(self, version: Union[str, _utils.Version]) -> bool:
         if isinstance(version, str):
-            version = _utils.Version.parse(version)
-            if not version:
+            version_obj = _utils.Version.parse(version)
+            if not version_obj:
                 return False
+            version = version_obj
         elif not isinstance(version, _utils.Version):
             return False
         # XXX Treat missing micro/release as wildcard?
-        return version.full == self.version.full
+        return version_obj.full == self.version.full
```

Comment:
I think I may have mixed something up in my last review.  Either we should use `version` or `version_obj`:

```python
            version = version_obj
        elif not isinstance(version, _utils.Version):
            return False
        # XXX Treat missing micro/release as wildcard?
        return version.full == self.version.full
```

or

```python
        elif not isinstance(version, _utils.Version):
            return False
        else:
            version_obj = version
        # XXX Treat missing micro/release as wildcard?
        return version_obj.full == self.version.full
```